packages/devtools_app/lib/src/memory/memory_filter.dart

```diff
@@ -229,7 +246,8 @@ class SnapshotFilterState extends State<SnapshotFilterDialog>
 
   Widget createLibraryListBox(BoxConstraints constraints) {
     return SizedBox(
-      height: constraints.maxHeight / 4,
+      height: constraints.maxHeight /
+          (controller.filteredLibrariesByGroupName.length - .5),
```

Comment:
can you explain this math?

Reply:
This was really old code that should use flex.  I've added a TODO to convert this to flex.